mod/metrics/lib/javascript/script_metrics.js.coffee

```diff
@@ -49,18 +72,40 @@ decko.editorContentFunctionMap['.wikiRating-editor'] = ->
 
 wikiRatingEditorHash = (table) ->
   hash = {}
-  table.find("tbody tr").each ->
+  variableMetricRows(table).each ->
     tr = $(this)
     if key = tr.find(".metric-label .thumbnail").data "cardName"
       hash[key] = tr.find(".metric-weight input").val()
-  hash
+  hash 
+
+# if all values are equals active the equalize
+activeEqualize = () -> 
+  if !getValuesFromTable( $('.wikiRating-editor') )  
```

Comment:
I would suggest something like this:
```
values = getValuesFromTable( $('.wikiRating-editor') )
$('#equalizer').prop 'checked', variableValuesAreEqual(values)
```
... why do we call toEqualize if the values are already equal?